src/modules/PowerOCR/PowerOCR/Helpers/WindowUtilities.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Windows;
 using System.Windows.Forms;
+using Microsoft.PowerToys.Telemetry;
```

Comment:
does this need installer update?

Reply:
probably not, just checking

Reply:
It did. That's why I did installer changes only after that.
Stills needs some changes now.